packages/commercetools/composables/src/useCart/index.ts

```diff
@@ -13,31 +15,55 @@ const getBasketItemByProduct = ({ currentCart, product }) => {
   return currentCart.lineItems.find((item) => item.productId === product._id);
 };
 
```

Comment:
For mouse-over explanation
```suggestion
/** returns current cart or creates new one **/
```